streamer_test.go

```diff
@@ -171,7 +171,7 @@ type testStreamerConfig struct {
 func newTestStreamer(ctx context.Context, t *testing.T, cfg testStreamerConfig) (*stubBQClient, *Streamer) {
```

Comment:
No need for some test cases in `streamer_test` or you feel it doesn't add any value?

Reply:
I've added some cases now that most of the defaults and the validation has moved to `streamer_config`